packages/model-viewer/src/three-components/PPRenderPipeline.ts

```diff
@@ -0,0 +1,30 @@
+import { EffectComposer, RenderPass, EffectPass, BloomEffect, BlendFunction, KernelSize, FXAAEffect, SSAOEffect, ToneMappingEffect, ToneMappingMode } from 'postprocessing';
+import { WebGLRenderer } from 'three';
+
+export const RENDER_PASS = new RenderPass(null, null);
+
+export const FXAA_EFFECT = new FXAAEffect();
+
+export const SSAO_EFFECT = new SSAOEffect();
+
+export const BLOOM_EFFECT = new BloomEffect({
+    blendFunction: BlendFunction.ADD,
+    mipmapBlur: true,
+    luminanceThreshold: 0.85,
+    luminanceSmoothing: 0.025,
+    intensity: 3,
+    kernelSize: KernelSize.LARGE
```

Comment:
Do any of these parameters need to be part of the public API? I haven't used bloom much; I love things being simple, but curious how much adjustment tends to be needed.

Reply:
It's definitely something to be considered, I too want the api to be kept to a minimum simplicity, when I'll make some tests I'd love for your input on how much of the parameters to expose in the api

Reply:
You can now take a look at all the effects I've implemented (mostly those I think would be useful, i.e outline for product showcases, bloom/ssao/color-grade/smaa for visual fidelity, glitch and pixelate for more niche users). I haven't exposed every parameter in most of the effects, only those I think would be useful to edit.